unittests/backends/iqm/IQMStartServerAndTest.sh.in

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# ============================================================================ #
+# Copyright (c) 2022 - 2023 NVIDIA Corporation & Affiliates.                   #
+# All rights reserved.                                                         #
+#                                                                              #
+# This source code and the accompanying materials are made available under     #
+# the terms of the Apache License 2.0 which accompanies this distribution.     #
+# ============================================================================ #
+
+# We'll use iqm-client tools to verify the request struct and generate responses
+@Python_EXECUTABLE@ -m pip install iqm-client --user
```

Comment:
I'd suggest removing this line and count on the developer installing iqm-client if they want to. build_cudaq.sh will produce an error message if the developer enables the mock server tests and they don't already have iqm-client installed, so I don't think we need this additional attempted installation here.
```suggestion
```

Reply:
The build of cudaq should not produce an error if the iqm-client is not installed (see my comment above). But I agree that tests should not install anything - the tests should/will instead be skipped when the iqm client is missing.

Reply:
Makes sense. However I think this is not enough to make tests pass/skip without `iqm-client` installed, it is not just pytest tests rely on `iqm-client`, but `IQMTester.cpp` and the mock server itself (running from the shell script). I think relying on a custom package to run the mock server brings too many issues and it would be better to remove it and make the mock server self-sufficient. Will do.

Reply:
Actually, found a way to mark IQM tests skipped from the `IQMStartServerAndTest.sh`. If there is no `iqm-client` installed the test is marked as skipped and does not even try to run mock server. Please check if this solution makes sense to you.

Reply:
Got it. Thank you - looks good to me.